pkg/draft/pack/create.go

```diff
@@ -104,7 +104,13 @@ func Create(name, dir string, files []*File) (string, error) {
 	// Next, we can simply loop through files and create each.
 	// We call MkdirAll for a safe way to create any missing dirs.
 	for _, f := range files {
-		fullpath := filepath.Join(path, f.Path)
+
+		// We need to strip the root path since we are re rooting
```

Comment:
this looks like a good change, but would you mind writing out a unit test or elaborate a little more on what this fixes?